packages/app/src/services/confetti.js

```diff
@@ -0,0 +1,141 @@
+import p5 from 'p5'
+
+/**
+ * A p5.js 'sketch' that renders the confetti pop animation.
+ * Further information about p5: https://p5js.org
+ * @param {*} sketch
+ */
+
+const confetti = (sketch) => {
+
+  let width = sketch.windowWidth
+  let height = sketch.windowHeight
+  let particles
+  let opacity
+  let gravity
+
+  let particleTypes = ['RECTANGLE', 'CIRCLE', 'TRIANGLE', 'LINE']
+
+  /**
+   * Init an array of confetti particles with:
+   * - The shape of the particle
+   * - A position (as a vector)
+   * - A target position / direction (as a vector)
+   * - A random color @see randomColor
+   * - Angle
+   * - Quasi-random rotation speed
+   */
+
+  const initParticles = () => {
```

Comment:
`@see` tags are not intended to be used like this. They should be at the end of the JSDoc comment, like this:
```suggestion
  /**
   * Init an array of confetti particles with:
   * - The shape of the particle
   * - A position (as a vector)
   * - A target position / direction (as a vector)
   * - A random color
   * - Angle
   * - Quasi-random rotation speed
   *
   * @see randomColor
   */
  const initParticles = () => {
```
If linking in-line is desired, `{@link targetName}` can be used.

For further reference: [`@link`](https://jsdoc.app/tags-inline-link.html), [`@see`](https://jsdoc.app/tags-see.html)